usbsdmux/__main__.py

```diff
@@ -13,20 +15,46 @@ def main():
         help="mode to switch to",
         choices=["dut", "host", "off", "client"],
         type=str.lower)
+    parser.add_argument(
+        "-c",
+        "--client",
+        help="Run in client mode with socket /tmp/sdmux.sock",
+        action="store_true",
+        default=False)
```

Comment:
Maybe we could default to client mode for non-root users?

Reply:
I added this feature in 791e514aff389d04d381862c4cdad3fa1df4234d. usbsdmux now tries to guess the best mode for the current user. Additionally a new switch is introduced to force usbsdmux into direkt mode.